benchmark/Microsoft.IdentityModel.Benchmarks/AntiVirusFriendlyConfig.cs

```diff
@@ -1,18 +1,97 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using BenchmarkDotNet.Columns;
 using BenchmarkDotNet.Configs;
 using BenchmarkDotNet.Jobs;
 using BenchmarkDotNet.Toolchains.InProcess.Emit;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+
 
 namespace Microsoft.IdentityModel.Benchmarks
 {
+        // Define custom columns for P50 and P99 latency
+    public class P50Column : IColumn
```

Comment:
https://benchmarkdotnet.org/articles/features/statistics.html

Most of these look like they already exist in StatisticColumn?

Reply:
Right. The columns explicitly get the ones we want and not the others